addon/templates/components/validated-input.hbs

```diff
@@ -99,8 +99,8 @@
   {{/if}}
 {{/if}}
 
-{{#if help}}
-  <p class={{config.css.help}}>{{help}}</p>
+{{#if hint}}
+  <p class={{config.css.hint}}>{{hint}}</p>
 {{/if}}
```

Comment:
Since renaming help to hint will introduce a breaking change anyway, I'd propose another one: Swapping the order of the hint and validation errors. I think it looks weird if the validation message appears below the hint, and the UX is probably better too (see [here](https://medium.com/@andrew.burton/form-validation-best-practices-8e3bec7d0549), "The best position for validation").